src/ipc/prepare-boot-usb.ts

```diff
@@ -6,15 +6,12 @@ import { RawDataReturn } from './get-usb-drives';
 export const channel = 'prepare-boot-usb';
 const debug = makeDebug('kiosk-browser:prepare-boot-usb');
 
-const BootableOptionPattern = /^Boot([0-9a-fA-F]+)\*?\s+(.+)\s+(.+)$/;
-const HardDriveDetailsPattern = /^HD\(.+,.+,(.+),.+,.+\).+$/;
+const BootableOptionPattern = /^Boot([0-9a-fA-F]+)\*?\s+(.+)$/;
```

Comment:
I wonder if it's worth making this `[0-9a-fA-F]\{4}`, since we know there will only ever be 4 digits

Reply:
also, the second capture group will grab everything (`Linpus Lite   	HD(1,MDR,0x314d08f,0x800,0x100000)/File(\\EFI\\boot\\grubx64.efi)RC`), if we wanted to just get the dev name instead of the EFI path you could split on tabs.

Reply:
Unless we think there's a chance we'd pick up something incorrect, I'd rather stay loose with the regexps, so we don't end up in a situation we didn't anticipate (are we sure it's only ever 4 digits, for example?)